privaterelay/templates/profile.html

```diff
@@ -90,12 +90,14 @@
       {% include "includes/banners.html" %}
 
 
-      {% if relay_addresses %}      
+      {% if relay_addresses or domain_addresses %}
         {% include "includes/dashboard-filter.html" %}
       {% endif %}
 
       {% if relay_addresses|length == 0 %}
-        {% include "includes/dashboard_onboarding.html" %}
+        {% if domain_addresses|length == 0 %}
```

Comment:
If there's a way to do conjunctions in combination with `| length`, let me know. (Yes, I miss JSX.)

Reply:
Oh! I think we can? https://docs.djangoproject.com/en/3.2/ref/templates/builtins/#boolean-operators

<img width="788" alt="image" src="https://user-images.githubusercontent.com/2692333/140172067-b759fa39-d65b-4339-babc-df7c8980cde8.png">

Reply:
Heh, for some reason I was under the impression that that would have to be done _before_ passing it to ` | length`, and was so convinced that I didn't even try it :P